src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -36,9 +37,12 @@
     //
     public static final String SAVED_FLAGS_FILENAME = "savedflags.dat";
     // See https://github.com/TotalFreedom/License - None of the listed names may be removed.
-    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ");
+    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "WickedGamingUK", "aggelosQQ", "OxLemonxO", "marcocorriero", "Commodore64x");
```

Comment:
Why you have removed me I do not know, this is a list of TFM developers, not TF developers. You are also missing the comment above I see that does say names should not be removed.